src/app/collection/collection.module.ts

```diff
@@ -1,10 +1,13 @@
-import { NgModule } from '@angular/core';
+import { CUSTOM_ELEMENTS_SCHEMA, NgModule } from '@angular/core';
 import { CommonModule } from '@angular/common';
 import { CollectionIndexComponent } from './pages/collection-index/collection-index.component';
 import { GenericPageComponent } from './pages/generic-page/generic-page.component';
 import { CuratorCardComponent } from './shared/included/curator-card/curator-card.component';
 import { StatCardComponent } from './shared/included/stat-card/stat-card.component';
 import { SecurityInjectionsComponent } from './pages/security-injections/security-injections.component';
+import { NiceChallengeModule } from './pages/nice-challenge/nice-challenge.module';
+import { CollectionsRoutingModule } from './collection.routing';
+import { NO_ERRORS_SCHEMA } from '@angular/core';
```

Comment:
This is just nit-picky; this could just be included in the first import statement since they are all from '@angular/core'